src/main/java/org/homeschoolpebt/app/submission/messages/ScheduledMessages.java

```diff
@@ -0,0 +1,68 @@
+package org.homeschoolpebt.app.submission.messages;
+
+import formflow.library.data.Submission;
+import formflow.library.email.MailgunEmailClient;
+import lombok.extern.slf4j.Slf4j;
+import org.homeschoolpebt.app.data.Transmission;
+import org.homeschoolpebt.app.data.TransmissionRepository;
+import org.homeschoolpebt.app.utils.SubmissionUtilities;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAdjuster;
+import java.util.List;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+
+@Slf4j
+@Component
+public class ScheduledMessages {
+  MailgunEmailClient mailgunEmailClient;
+  TwilioSmsClient twilioSmsClient;
+  private final TransmissionRepository transmissionRepository;
+  static final TemporalAdjuster REMINDER_TIME_FRAME = t -> t.minus(2, DAYS);
+
+  public ScheduledMessages(TransmissionRepository transmissionRepository) {
+    this.transmissionRepository = transmissionRepository;
+  }
+
+  @Scheduled(cron = "0 0 19 * * *") // every day at 7pm GMT/Noon PDT
```

Comment:
How will this work with multiple containers running? It'd be bad if they all decided to send messages simultaneously. It looks like there's some library called `ShedLock` for this purpose: https://www.baeldung.com/shedlock-spring

Since I don't know how that works (and therefore I don't trust it), I was hoping to accomplish cron for the transmitter via a more bash-y approach as we do on GCF. For that, we have a `crontab` file and run it with [supercronic](https://github.com/aptible/supercronic).

See `app/cli/TransmitterCommands.java` which defines a command that can be run with:

```
java -jar [build.jar] -Dspring.main.web-application-type=NONE transmit
```

I think the change here would be minimal - we can remove all the Spring scheduling and just call `checkForUnsubmittedDocs` in a CLI method instead? How does that sound?

Reply:
Sure! I was going more along those lines at first but wasn't sure if there was a need for supercronic. I meant to ask if `@Scheduled` was sufficient, but you've already answered :)